src/picongpu/include/simulation_defines/param/memory.param

```diff
@@ -56,4 +56,11 @@ namespace picongpu
     /** number of scalar fields that are reserved as temporary fields */
     constexpr uint32_t fieldTmpNumSlots = 1;
 
+    /** can `FieldTmp` gather neighbor information
+     *
+     * If `true` it is possible to call the method `asyncCommunicationGather()`
+     * to copy data from the border of neighboring GPU into the local guard.
+     */
```

Comment:
forgot: 
* This is also known as building up a "ghost" or "halo" region in domain
* decomposition and only necessary for specific algorithms that extend
* the basic PIC cycle, e.g. with dependence on derived density or energy fields.